deployment/CP05MOAS-GL339_Deploy.csv

```diff
@@ -52,4 +52,10 @@ TN20181209,,WJ20190228,,CP05MOAS-GL339-01-ADCPAM000,9,1,2018-12-09T16:36:49,2019
 TN20181209,,WJ20190228,,CP05MOAS-GL339-02-FLORTM000,9,1,2018-12-09T16:36:49,2019-03-01T14:19:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-FLORTM-03190,40.082183,-70.1885,,200,200,EB
 TN20181209,,WJ20190228,,CP05MOAS-GL339-03-CTDGVM000,9,1,2018-12-09T16:36:49,2019-03-01T14:19:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-CTDGVM-09083,40.082183,-70.1885,,200,200,EB
 TN20181209,,WJ20190228,,CP05MOAS-GL339-04-DOSTAM000,9,1,2018-12-09T16:36:49,2019-03-01T14:19:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-DOSTAM-00566,40.082183,-70.1885,,200,200,EB
-TN20181209,,WJ20190228,,CP05MOAS-GL339-05-PARADM000,9,1,2018-12-09T16:36:49,2019-03-01T14:19:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50203,40.082183,-70.1885,,200,200,EB
\ No newline at end of file
+TN20181209,,WJ20190228,,CP05MOAS-GL339-05-PARADM000,9,1,2018-12-09T16:36:49,2019-03-01T14:19:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50203,40.082183,-70.1885,,200,200,EB
+TS20200301,,,,CP05MOAS-GL339-00-ENG000000,10,1,2020-03-02T11:09:59,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGCON-GLDRCP-00339,40.08131,-70.372867,,200,200,EB
+TS20200301,,,,CP05MOAS-GL339-01-ADCPAM000,10,1,2020-03-02T11:09:59,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-ADCPAM-72777,40.08131,-70.372867,,200,200,EB
+TS20200301,,,,CP05MOAS-GL339-02-FLORTM000,10,1,2020-03-02T11:09:59,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-FLORTM-04467,40.08131,-70.372867,,200,200,EB
+TS20200301,,,,CP05MOAS-GL339-03-CTDGVM000,10,1,2020-03-02T11:09:59,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-CTDGVM-09363,40.08131,-70.372867,,200,200,EB
+TS20200301,,,,CP05MOAS-GL339-04-DOSTAM000,10,1,2020-03-02T11:09:59,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-DOSTAM-00361,40.08131,-70.372867,,200,200,EB
+TS20200301,,,,CP05MOAS-GL339-05-PARADM000,10,1,2020-03-02T11:09:59,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50238,40.08131,-70.372867,,200,200,EB
```

Comment:
I get 40.08132 lat

Reply:
I agree with 40.08132! I will change that now